src/main/kotlin/com/strumenta/kolasu/model/Printing.kt

```diff
@@ -9,31 +9,32 @@ private const val indentBlock = "  "
 fun Node.relevantMemberProperties() = this.javaClass.kotlin.memberProperties
     .filter { !it.name.startsWith("component") && it.name != "position" && it.name != "parent" }
 
+@Suppress("UNCHECKED_CAST")
 fun Node.multilineString(indent: String = ""): String {
     val sb = StringBuffer()
     if (this.relevantMemberProperties().isEmpty()) {
         sb.append("$indent${this.javaClass.simpleName}\n")
     } else {
         sb.append("$indent${this.javaClass.simpleName} {\n")
-        this.relevantMemberProperties().forEach {
-            val mt = it.returnType.javaType
-            if (mt is ParameterizedType && mt.rawType.equals(List::class.java)) {
+        this.relevantMemberProperties().forEach { property ->
+            val mt = property.returnType.javaType
+            if (mt is ParameterizedType && mt.rawType == List::class.java) {
```

Comment:
right!

Reply:
Yeah, I couldn't read the code so I explicitly called things "property" and "value" after a while. What's an "mt"? ;-)

The equals was just sitting there in the code, all yellow.